framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3318,6 +3340,45 @@ private AnnotationMirror getDeclAnnotation(
         return results;
     }
 
+    /**
+     * Adds into {@code results} the declaration annotations found in all elements of the super
```

Comment:
Should this say it adds the declaration annotations that are marked to be inherited?